mne_bids/bids_meg.py

```diff
@@ -29,24 +29,29 @@ def _channel_tsv(raw, fname):
 
     map_chs = dict(grad='MEG', mag='MEG', stim='TRIG', eeg='EEG',
                    eog='EOG', misc='MISC')
-    map_desc = dict(grad='sensor gradiometer', mag='magnetometer',
-                    stim='analogue trigger',
-                    eeg='electro-encephalography channel',
-                    eog='electro-oculogram', misc='miscellaneous channel')
+    map_desc = dict(grad='Gradiometer', mag='Magnetometer',
+                    stim='Trigger',
+                    eeg='ElectroEncephaloGram',
+                    ecg='ElectroCardioGram',
```

Comment:
okay looks like you misunderstood me and I misunderstood you :) I was saying that you have to update `map_chs` and add a key for ECG. I think you understood that I was asking you to change grad=MEG to grad=MAG !